### DISCUSSION TITLE
How to change default font of the grapejs editor.

### DISCUSSION THREAD

**Discussion by rafaelspereira1** [GrapesJS/grapesjs#5639] [Q&A]
Hi, i am trying to change the  default of the canvas that the editor uses but i am unable to find something in the docs about it. Is there a way to change it to another font?

**Comment by artf** [comment]
Similar to the [template scripts](https://grapesjs.com/docs/modules/Components-js.html#template-related)

**Comment by hongminpark** [comment]
@rafaelspereira1 Did you find solution ?
